prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -0,0 +1,116 @@
+# Make sure we can open sufficiently many files / allocate enough memory
+ulimit -n 4096 || ulimit -l unlimited || ulimit -m unlimited || ulimit -l unlimited
+if [ $? != 0 ]; then
+  echo Error setting ulimits
+  exit 1
+fi
+
+if [ "0$ALIENVLVL" == "0" ]; then
+  eval "`alienv shell-helper`"
+  alienv --no-refresh load O2/latest
+fi
+
+if [ -z "$NTIMEFRAMES" ];   then export NTIMEFRAMES=1; fi              # Number of time frames to process
```

Comment:
I think these lines could be (optionally) made more compact.
`export NTIMEFRAMES=${NTIMEFRAMES:-1}`